R/helpers.R

```diff
@@ -799,18 +799,36 @@ cell_style_to_html.cell_text <- function(style) {
 #'
 #' @param color The fill color. If nothing is provided, then `"#D3D3D3"` (light
 #'   gray) will be used as a default.
+#' @param alpha The alpha transparency value for the `color` as single value in
+#'   the range of `0` (fully transparent) to `1` (fully opaque). If not provided
+#'   the fill color will be fully opaque.
 #'
 #' @family helper functions
 #' @export
-cell_fill <- function(color = NULL) {
+cell_fill <- function(color = NULL,
+                      alpha = NULL) {
 
-  # Get all assigned values for the functions' arguments
-  style_names <- formals(cell_fill) %>% names()
-  style_names <- mget(style_names)
+  if (is.null(color)) {
+    color <- "#D3D3D3"
+  }
 
-  # Filter list by only the non-NULL (e.g., assigned with
-  # value) elements
-  style_vals <- style_names %>% .[!vapply(., is.null, logical(1))]
+  if (is.null(alpha)) {
+    alpha <- 1
+  }
+
+  # Get hexadecimal representation of `color`
+  color_hex <- color %>% get_hexcolor_vec()
+
+  if (color %>% is_hex_col_w_alpha()) {
+    alpha_hex <- color %>% get_alpha_vec()
+  } else {
+    alpha_hex <- alpha %>% frac_alpha_to_hex()
+  }
+
+  # Combine hexadecimal color with corresponding alpha
+  color_val <- color_hex %>% paste_right(alpha_hex)
```

Comment:
same issue as above.  make helper function of `color_with_default_alpha(color, alpha)`

Reply:
Great idea. I made the `normalize_color()` util function.